packages/transformers/postcss/src/PostCSSTransformer.js

```diff
@@ -79,10 +77,22 @@ export default (new Transformer({
     let cssModules: ?{|[string]: string|} = null;
     if (config.hydrated.modules) {
       asset.meta.cssModulesCompiled = true;
+
+      // TODO: should this be resolved from the project root?
```

Comment:
actually a question for all dev deps. What if there were a CSS module in node_modules? we would never want to install dev deps there...